src/main/java/io/vertx/core/http/impl/HttpServerImpl.java

```diff
@@ -638,14 +638,14 @@ protected void handleMessage(ServerConnection conn, ContextImpl context, Channel
           case BINARY:
           case CONTINUATION:
           case TEXT:
+          case PONG:
             conn.handleMessage(msg);
             break;
           case PING:
             // Echo back the content of the PING frame as PONG frame as specified in RFC 6455 Section 5.5.2
-            ch.writeAndFlush(new WebSocketFrameImpl(FrameType.PONG, wsFrame.getBinaryData()));
-            break;
-          case PONG:
```

Comment:
On the server, pong is moved so that the pongHandler is invoked.

Currently, the webSocketClient api exposes a pongHandler method, but the client does invoke it as the server does.  I would like to resolve the inconsistency, and request input on whether the client should expose that method.